### PR TITLE
OCS Power BI Connector review for GA release

### PR DESCRIPTION
**Pull request by rmuolic**

### CODE REVIEW

File: content/visualize-data/power-bi-connector.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+uid: PowerBI
+---
+
+# OSIsoft Cloud Services Power BI Connector
+
+The OSIsoft Cloud Services Power BI Connector retrieves data views from OCS and makes them available in Microsoft Power BI for advanced data visualization and analysis. Install and retrieve data views with the connector for use with Microsoft Power BI as described in this section.<br>
```
Comment (dwicinas-osisoft) [inline content/visualize-data/power-bi-connector.md:7]: I find that second sentence pretty unclear.  Which connector? Are you saying the user can install and retrieve data views?

File: content/visualize-data/power-bi-connector.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        - Open a command prompt window and type following command:<br>
+        `.\OCSDataConnectorInstaller.exe -Y INSTALLDIR="<install path>" /quiet`<br>
+
+             **Note:** OSIsoft Cloud Services Power BI Connector supports silent installation for on-premises data gateway installations.
+
+## Retrieve data views with OSIsoft Cloud Services Power BI Connector
+
+Use OSIsoft Cloud Services Power BI Connector to retrieve OCS data views for use in Microsoft Power BI.
+
+1. In Microsoft Power BI Desktop, in the entry field of the `Get Data` window, type *OSI*.<br>The OSIsoft Cloud Services Data Views connector file displays in the `All` pane.
```
Comment (dwicinas-osisoft) [inline content/visualize-data/power-bi-connector.md:31]: In this command you are typing a command in one fonts - *OSI*. On line 23 you are typing another command delineated by the tic mark (`). Is this inconsistent?

File: content/visualize-data/power-bi-connector.md (hunk 1 above)
Comment (rmuolic) [reply content/visualize-data/power-bi-connector.md:7]: > I find that second sentence pretty unclear. Which connector? Are you saying the user can install and retrieve data views?

It is the connector mentioned in the previous sentence, the OSIsoft Cloud Services Power BI Connector. Will rephrase the sentence to something like, "Install OSIsoft Cloud Services Power BI Connector to retrieve data views for use with Microsoft Power BI as described in this section."

File: content/visualize-data/power-bi-connector.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Click **Continue** in the `Connecting to a third-party service` warning.
+
+1. In the `OSIsoft Cloud Services Data Views` window, enter the OCS tenant and namespace for the data views you want to access, and then click **OK**.<br>The OSIsoft Cloud Services sign-in window displays if you have not already signed into OCS.
+
+    - If you have not already signed into OCS, click **Sign in as different user** and complete the OCS user authentication process.
+
+1. In the `OSIsoft Cloud Services Data Views` sign-in window, click **Connect**.<br>The `Navigator` pane displays.
+
+1. In the `Navigator` pane, click on either Interpolated Data Views or Stored Data Views, and select the data view you want to work with in Microsoft Power BI.
```
Comment (dwicinas-osisoft) [inline content/visualize-data/power-bi-connector.md:45]: Should the things you're clicking be in bold?

File: content/visualize-data/power-bi-connector.md (hunk 2 above)
Comment (rmuolic) [reply content/visualize-data/power-bi-connector.md:31]: > In this command you are typing a command in one fonts - _OSI_. On line 23 you are typing another command delineated by the tic mark (`). Is this inconsistent?

I will update to `OSI`. Thank you.

File: content/visualize-data/power-bi-connector.md (hunk 3 above)
Comment (rmuolic) [reply content/visualize-data/power-bi-connector.md:45]: > Should the things you're clicking be in bold?

I will update Interpolated Data Views and Stored Data Views to bold.

File: content/visualize-data/power-bi-connector.md (hunk 1 above)
Comment (RVerhoeff) [reply content/visualize-data/power-bi-connector.md:7]: I see that the term "data views" is lower case throughout unless it's part of the name of the Connector product. Should we treat the term as a proper noun and use mixed case, as in "Data Views?" This is how Data Views appears in the OCS portal.

Comment (rmuolic) [reply content/visualize-data/power-bi-connector.md:7]: > I see that the term "data views" is lower case throughout unless it's part of the name of the Connector product. Should we treat the term as a proper noun and use mixed case, as in "Data Views?" This is how Data Views appears in the OCS portal.

Let's treat the term as a proper noun and use "Data Views". I can update throughout. Thank you.

**Comment by rmuolic** [comment]
Closed this PR and imported the finalized files for the Power BI Connector to the [rmuolic-power-bi-connector-ga branch](https://github.com/osisoft/OCS-Docs/tree/rmuolic-power-bi-connector-ga), see [PR 648](https://github.com/osisoft/OCS-Docs/pull/648).

**Pull request closed without merging**